### DISCUSSION TITLE
The documentation for version 3.0.0 contains two inconsistent descriptions of neuron

### DISCUSSION THREAD

**Discussion by 96XinHe** [deepmodeling/deepmd-kit#4470] [General]
The first place say : "...When two layers are of the **same size** **or** one layer is twice as large as the previous layer, a skip connection is built."  (https://docs.deepmodeling.com/projects/deepmd/en/v3.0.0/train/train-input.html#model-standard-descriptor-se-e2-a-neuron)

But another place says: "The [neuron](https://docs.deepmodeling.com/projects/deepmd/en/v3.0.0/train/train-input.html#model-standard-descriptor-se-e2-a-neuron) specifies the size of the embedding net. From left to right the members denote the sizes of each hidden layer from the input end to the output end, respectively. If the outer layer **is twice the size of the inner layer**, then the inner layer is copied and concatenated, then a [ResNet architecture] is built between them." (https://docs.deepmodeling.com/projects/deepmd/en/v3.0.0/model/train-se-e2-a.html).

Inconsistent expression of whether or not ResNet connections can be established between two layers of the **same size**. Maybe one of the descriptions should be revised？

**Comment by njzjz** [comment]
The first one is correct.

**Reply by wanghan-iapcm** [reply]
both of them are correct. the first one may mention the concatenation of the input.

**Reply by njzjz** [reply]
I believe we should copy the equations of the neural networks from the paper to somewhere in the documentation..
